lib/node_modules/@stdlib/number/float64/to-json/lib/main.js

```diff
@@ -0,0 +1,69 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
```

Comment:
```suggestion
* Copyright (c) 2022 The Stdlib Authors.
```

Reply:
Sorry for this, I will update that :)

Reply:
No worries!

@Planeshifter Maybe we can create some tooling which (a) lints the copyright year for new files and (b) auto-corrects this for PRs adding new files. A combination of ESLint and ESlint fix.